pkg/rancher-desktop/window/index.ts

```diff
@@ -616,3 +618,23 @@ export function send(channel: string, ...args: any[]) {
     }
   }
 }
+
+/**
+ * Center the dialog window in the middle of parent window - used on Windows / MacOs
+ * @param window parent window
+ * @param dialog dialog window
+ * @param offsetX
+ * @param offsetY
+ */
+export function centerDialog(window: BrowserWindow, dialog: BrowserWindow, offsetX = 0, offsetY = 0) {
+  const windowBounds = window.getBounds();
+  const dialogBounds = dialog.getBounds();
+
+  console.log(windowBounds);
+  console.log(dialogBounds);
```

Comment:
We should provide more information about what this is logging if we need these statements. Otherwise, we can just remove them

Reply:
Let's remove them.